src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+
+      <p className="Person__age">
+        {`I am ${person.age || null}`}
+      </p>
```

Comment:
do you need a `p` with a `null` value? I don't think so. You can wrap your `p` tag with `age &&` condition